nodebrew

```diff
@@ -372,6 +372,7 @@ sub _cmd_exec {
     my $command = join ' ', @$args;
 
     system $command;
+    exit $? >> 8;
```

Comment:
If `$command;` exit with `1`, `system` method return `256`. 
It is needed to shift right by `8` for getting actual exit value.  


>The return value is the exit status of the program as returned by the wait call. To get the actual exit value, shift right by eight (see below). 

http://perldoc.perl.org/functions/system.html